src/components/selectInput/selectInput.css

```diff
@@ -0,0 +1,83 @@
+*{
+    font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
dont use * 
this will affect all the other components use 
Roboto is already added to the repo